### ISSUE TITLE
upgrade error

### ISSUE THREAD

**Issue by ildar** [thuryn/your-local-weather#83]
after last upgrade through F-Droid YLW constantly crashing on startup:
```
10-23 11:25:11.812  5632  5707 I ActivityManager: Displayed org.thosp.yourlocalweather/.MainActivity: +512ms
10-23 11:25:12.666  2975  2975 E AndroidRuntime: FATAL EXCEPTION: main
10-23 11:25:12.666  2975  2975 E AndroidRuntime: Process: org.thosp.yourlocalweather, PID: 2975
10-23 11:25:12.666  2975  2975 E AndroidRuntime: java.lang.RuntimeException: java.lang.NumberFormatException: For input string: "OFF"
10-23 11:25:12.666  2975  2975 E AndroidRuntime: 	at android.app.job.JobService$JobHandler.handleMessage(JobService.java:147)
10-23 11:25:12.666  2975  2975 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:102)
10-23 11:25:12.666  2975  2975 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:154)
10-23 11:25:12.666  2975  2975 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6186)
10-23 11:25:12.666  2975  2975 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
10-23 11:25:12.666  2975  2975 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
10-23 11:25:12.666  2975  2975 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
10-23 11:25:12.666  2975  2975 E AndroidRuntime: 	at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:107)
10-23 11:25:12.666  2975  2975 E AndroidRuntime: Caused by: java.lang.NumberFormatException: For input string: "OFF"
10-23 11:25:12.666  2975  2975 E AndroidRuntime: 	at java.lang.Integer.parseInt(Integer.java:521)
10-23 11:25:12.666  2975  2975 E AndroidRuntime: 	at java.lang.Integer.parseInt(Integer.java:556)
10-23 11:25:12.666  2975  2975 E AndroidRuntime: 	at org.thosp.yourlocalweather.utils.Utils.intervalMillisForAlarm(Utils.java:492)
10-23 11:25:12.666  2975  2975 E AndroidRuntime: 	at org.thosp.yourlocalweather.service.AbstractAppJob.reScheduleNextAlarm(AbstractAppJob.java:44)
10-23 11:25:12.666  2975  2975 E AndroidRuntime: 	at org.thosp.yourlocalweather.service.StartAppAlarmJob.onStartJob(StartAppAlarmJob.java:24)
10-23 11:25:12.666  2975  2975 E AndroidRuntime: 	at android.app.job.JobService$JobHandler.handleMessage(JobService.java:143)
10-23 11:25:12.666  2975  2975 E AndroidRuntime: 	... 7 more
10-23 11:25:12.671  5632  9993 W ActivityManager:   Force finishing activity org.thosp.yourlocalweather/.MainActivity
```

**Comment by thuryn** [comment]
Sorry, fixed in 4.3.2 version

**Comment by ildar** [comment]
allright, thanks. I'll hardly test it as I wiped data already.
